doc/manual/manual.tex

```diff
@@ -320,11 +320,12 @@ \subsection{Referencing \aspect{}}
 }
 \end{lstlisting}
 
+and cIte the  most recent release of  the software package. See \url{https://aspect.dealii.org/publications}.
```

Comment:
Depends on where you think the sentence starts. Will fix the dropped portion of the url.